pubsub/gcp/pubsub/pubsub.go

```diff
@@ -82,6 +83,10 @@ type WhatNow struct {
 	Type string `json:"type"`
 }
 
+type metadata struct {
+    AckDeadline int `mapstructure:"ackDeadline"`
```

Comment:
Please use type as `time.Duration` so Dapr can be smart and interpret it as seconds or Go durations

Reply:
Thank you for your feedback @ItalyPaleAle, I've updated the type to `time.Duration`